lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -985,6 +999,8 @@ def get_proj(self):
         point.
 
         """
+        pb_aspect = np.array([1, 1, 0.75])
```

Comment:
Strictly speaking, this isn't just the aspect, but also the scaling. Is there already scaling in place somewhere else?